setup.sh

```diff
@@ -0,0 +1,25 @@
+!/bin/bash
+
+Run the go_installer script with the parent directory as an argument
+./go_installer.sh ../
+echo "Installed go" 
+go version
+sudo apt update -y
```

Comment:
I think this script cannot be used by the distros which are not ubuntu

Reply:
Yes we mostly use ubuntu Vm other than that very rarely we may use rhel to recreate issues. Can add script for rhel as well if its useful!

Reply:
Mentioned its for ubuntu machines only via a comment